Autopilot/SensorFusion/Src/SensorFusion.cpp

```diff
@@ -75,6 +92,36 @@ void localToGlobalAccel(IMUData_t *imudata, float *u)
     u[2] = imudata->accz; //Longitudinal acceleration
 }
 
+//Map gps position to xy coords using the reference location as the origin.
+//wikipedia.org/wiki/Geographic_coordinate_system
+double * gpsToCartesian(double lat, double lng){
+    const double RAD_LAT = REF_LAT*ZP_PI/180;
```

Comment:
MathConstants.hpp already has these buil in
![image](https://user-images.githubusercontent.com/32182266/122675600-23302200-d1a8-11eb-9b0e-170491cee5b7.png)
You should use them instad of doing the conversion manually like this